src/exposurescrawler/crawlers/tableau.py

```diff
@@ -18,6 +18,12 @@
 
 
 def _should_ignore_workbook(workbook, projects_to_ignore: Collection[str]) -> bool:
+    # Personal spaces are usually used as a sandbox for experimental work
```

Comment:
Would be nice to add a test for this.
We do have some tests in this repo, but I think we can improve.
Let me know if you would like to do it together.

Reply:
@kevinneville I merged this PR. I guess your comment was added after the merge. We can do it together sometime soon. :)